site/src/pages/HealthPage/DERPPage.tsx

```diff
@@ -0,0 +1,122 @@
+import { Link, useOutletContext } from "react-router-dom";
+import {
+  Header,
+  HeaderTitle,
+  Main,
+  SectionLabel,
+  BooleanPill,
+  Logs,
+  HealthyDot,
+} from "./Content";
+import { HealthcheckReport } from "api/typesGenerated";
+import Button from "@mui/material/Button";
+import LocationOnOutlined from "@mui/icons-material/LocationOnOutlined";
+import useTheme from "@mui/styles/useTheme";
+import { healthyColor } from "./healthyColor";
+import { Alert } from "components/Alert/Alert";
+import { Helmet } from "react-helmet-async";
+import { pageTitle } from "utils/page";
+
+const flags = [
+  "UDP",
+  "IPv6",
+  "IPv4",
+  "IPv6CanSend",
+  "IPv4CanSend",
+  "OSHasIPv6",
+  "ICMPv4",
+  "MappingVariesByDestIP",
+  "HairPinning",
+  "UPnP",
+  "PMP",
+  "PCP",
+];
+
+export const DERPPage = () => {
+  const { derp } = useOutletContext<HealthcheckReport>();
+  const { netcheck, regions, netcheck_logs: logs } = derp;
+  const theme = useTheme();
+
+  return (
+    <>
+      <Helmet>
+        <title>{pageTitle("DERP - Health")}</title>
+      </Helmet>
+
+      <Header>
+        <HeaderTitle>
+          <HealthyDot
+            healthy={derp.healthy}
+            hasWarnings={derp.warnings.length > 0}
+          />
+          DERP
+        </HeaderTitle>
+      </Header>
+
+      <Main>
+        {derp.warnings.map((warning, i) => {
+          return (
+            <Alert key={i} severity="warning">
```

Comment:
Flagging this with the same message as the other Alert case, just in case you do think that the keys should get swapped out